README.md

```
@@ -1,66 +1,70 @@
-mxlrcsvc-go example_input.txt -c 20
+```sh
+mxlrcgo-svc example_input.txt -c 20
 ```
 ### Directory Mode (recursive)
-```
-mxlrcsvc-go "Dream Theater"
+```sh
+mxlrcgo-svc "Dream Theater"
 ```
 > **_This option overrides the `-o/--outdir` argument which means the lyrics will be saved in the same directory as the given input._**
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)